tests/test_client.py

```diff
@@ -322,7 +361,7 @@ def test_get_buy_url_contribution(self):
         )
         url = self.lp.get_buy_url(item)
         self.assertTrue(url.startswith('https://web.laterpay.net/dialog/contribute/pay_now?'))
-        self.assertQueryString(url, 'campaign_id', value='2')
+        self.assertQueryString(url, 'campaign_id', value='save-the-world')
```

Comment:
`'save-the-world'` is the correct value. However, the tests pass, regardless of what the value is here. Something isn't right.